examples/gno.land/r/gnoland/hof/hof.gno

```diff
@@ -0,0 +1,174 @@
+// Package hof is the gno.land hall of fame realm.
+// It contains a permanent exhibition of items in the hall of fame,
+// a possibility to host temporary ones created by its admins.
+package hof
+
+import (
+	"std"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/seqid"
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	exhibCounter seqid.ID
+	exhibitions  *avl.Tree // id > *Exhibition
```

Comment:
i've a doubt, shoudl `avl.Tree` be a pointer?

Reply:
can be both as the methods use a pointer receiver, which can be "implicit". (same as declaring `var buf bytes.Buffer` and then using `buf.Write`).

I guess the advantage to not making it a pointer is that the zero value is ready to use.

Reply:
`NewTree` returns a pointer; we should decide there first.